frontend/src/contexts/AppContext.tsx

```
@@ -0,0 +1,167 @@
+    case 'SET_USER':
+      return { ...state, user: action.payload };
+    case 'SET_AUTHENTICATED':
+      return { ...state, isAuthenticated: action.payload };
+    case 'SET_CURRENT_SESSION':
+      return { ...state, currentSession: action.payload };
+    case 'SET_MESSAGES':
+      return { ...state, messages: action.payload };
+    case 'ADD_MESSAGE':
+      return { ...state, messages: [...state.messages, action.payload] };
+    case 'UPDATE_MESSAGE':
+      const updatedMessages = [...state.messages];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)